src/components/pages/txs/TransactionTable.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import Link from "next/link";
+import { formatEther, formatGwei } from "viem";
+
+function formatTimestamp(timestamp: number): string {
+  const date = new Date(timestamp * 1000);
+  const now = new Date();
+  const diffTime = Math.abs(now.getTime() - date.getTime());
+  const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+
+  return `${diffDays} day${diffDays !== 1 ? 's' : ''} ago`;
+}
+
+export function TransactionsTable({ transactions }: { 
+transactions: any }) {
+  return (
+    <Table>
+      <TableHeader>
+        <TableRow>
+          <TableHead>Tx Hash</TableHead>
+          <TableHead>Method ID</TableHead>
+          <TableHead>Block</TableHead>
+          <TableHead>Age</TableHead>
+          <TableHead>From</TableHead>
+          <TableHead>To</TableHead>
+          <TableHead>Value</TableHead>
+          <TableHead>Txn Fee</TableHead>
+        </TableRow>
+      </TableHeader>
+      <TableBody>
+        {transactions.map((tx: any) =>{
+           const formattedtimestamp = formatTimestamp(Number(tx.timestamp));
+           return(
+          <TableRow key={tx.hash}>
+            <TableCell>
+              <Link
+                href={`/tx/${tx.hash}`}
+                className="text-blue-500 hover:underline"
```

Comment:
Make sure to adhere to the same look and feel of the current app: this should be `text-primary hover:brightness-150` instead.